pkg/securelaunch/helpers.go

```diff
@@ -12,26 +12,45 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"sync"
 
-	"github.com/u-root/u-root/pkg/boot/diskboot"
+	"github.com/u-root/u-root/pkg/mount"
 	"github.com/u-root/u-root/pkg/storage"
 )
 
-/* used to store all block devices returned from a call to storage.GetBlockStats */
-var storageBlkDevices []storage.BlockDev
+type persistDataItem struct {
+	desc        string // Description
+	data        []byte
+	location    string // of form sda:/path/to/file
+	defaultFile string // if location turns out to be dir only
+}
+
+var persistData []persistDataItem
+
+type mountCacheData struct {
+	flags     uintptr
```

Comment:
why did you use `uintptr` as the type? this type is used as an unsigned int version of a memory pointer. i would recommend just using int or even uint32.

Reply:
This type needs to match flags variable type that is passed to device.Mount function. We compare flags so types need to match...